manifests/m/MicroSIP/MicroSIP-Lite/3.20.7/MicroSIP.MicroSIP-Lite.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: MicroSIP.MicroSIP-Lite
```

Comment:
```suggestion
PackageIdentifier: MicroSIP.MicroSIP.Lite
```

Reply:
Same here as in MicroSIP.MicroSIP-Lite.locale.en-US.yaml.

Reply:
Resolved in 0725fd5eb4a72647ddc134db1b2cbeda594beeb9